x-pack/plugins/ml/public/application/components/ml_inference/components/test_pipeline.tsx

```diff
@@ -35,32 +39,56 @@ import { useMlApiContext, useMlKibana } from '../../../contexts/kibana';
 import { getPipelineConfig } from '../get_pipeline_config';
 import { isValidJson } from '../../../../../common/util/validation_utils';
 import type { MlInferenceState } from '../types';
+import { checkIndexExists } from '../retry_create_data_view';
+
+const sourceIndexMissingMessage = i18n.translate(
+  'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.sourceIndexMissing',
+  {
+    defaultMessage:
+      'Source index is missing. Unable to fetch sample documents. Please add sample documents.',
```

Comment:
Suggested edit - `The source index used to train the model is missing. Enter the text for documents to test the pipeline below.`

Any thoughts @szabosteve ?

Reply:
Or it can be even a bit shorter:

```
The source index used to train the model is missing. Enter text for documents to test the pipeline.
```

Reply:
Updated in 2c51898d8c7022455d87e5815e92eb7880e092f9